Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -130,10 +130,18 @@ void StackingContext::paint_descendants(PaintContext& context, Layout::Node cons
 
 void StackingContext::paint_internal(PaintContext& context) const
 {
-    // For a more elaborate description of the algorithm, see CSS 2.1 Appendix E
-    // Draw the background and borders for the context root (steps 1, 2)
-    paint_node(m_box, context, PaintPhase::Background);
-    paint_node(m_box, context, PaintPhase::Border);
+    // helpers
+
```

Comment:
We usually don't have an empty line to start a function

Reply:
removed it